4_pokemonpage/css-basics_pokemon-page/css/styles.css

```diff
@@ -3,15 +3,15 @@
 /* ------- General Styles ------- */
 
 /* Task 1: First of all we want to adjust how the size of our elements is calculated when working with CSS. Do you know how to switch from "content-box" to "border-box"? */
-*{
-  box-sizing: border-box;
-}
+
+
 /* your solution */
 
 /* Task 2: Let's work on some basic stylings that we want to apply to all elements of your code via inheritance. Which selector do we need for this? */
 
 /* add selector */
 * {
```

Comment:
How about "html" or "body" instead of "*"? (Since it seems it should be 2 different selectors)

Reply:
What would speak for html instead of * and having two separate selectors?
If it's just the challenge, i'd like to keep it in *. 😃